src/content/docs/guides/using-hardhat.md

```diff
@@ -0,0 +1,152 @@
+---
+title: Deploy a contract using Hardhat
+description: This guide will help you deploy a smart contract on Taiko using Hardhat.
+---
+
+Hardhat is an Ethereum development environment for deploying smart contracts, running tests, and debugging Solidity code locally. It is one of the popular smart contract development frameworks. This guide demonstrates deploying a `Storage smart contract` on Taiko using Hardhat, highlighting the compatibility of Ethereum contracts with Taiko. 
+
+For a visual walkthrough, refer to our [video tutorial](https://drive.google.com/file/d/1HSTlu0NsPtDl7IyYk9cDtU3yEnyEbLxL/view?usp=sharing).
+
+## Set up the environment
+Hardhat is a flexible Ethereum development environment designed for testing, compiling, and deploying smart contracts. It requires Node.js, npm, and Git to get started.
+
+To effectively use Hardhat, your system must have [Node.js](https://nodejs.org/en/) (v10+ LTS) and npm installed. 
+
+:::tip
+If you are using Windows, we strongly recommend using [WSL 2](https://learn.microsoft.com/en-us/assessments/) to follow this guide.
+:::
+
+## Create a Hardhat project
+- In your desired directory, run
+```bash
+mkdir hello-hardhat && cd hello-hardhat
+```
+- Initialize an npm project
+```bash
+npm init -y
+```
+- Install dotenv for environment variable management
+```bash
+npm install dotenv
+```
+- Add Hardhat to your project
+```bash
+npm install --save-dev hardhat
+```
+- Initialize Hardhat
+```bash
+npx hardhat init
+```
+Choose `Create a JavaScript project` and follow the prompts, agreeing to create a `.gitignore` and install suggested dependencies.
+
+## Configure Your Project
+
+- Delete `deploy.js` in the `/scripts` directory and `Lock.sol` in the `/contracts` directory.
```

Comment:
`npx hardhat init` no longer generates deploy scripts when following the Javascript project prompts